docs/source/content/unsupervised/pca.rst

```diff
@@ -0,0 +1,117 @@
+Principal Component Analysis
+============================
+
+.. contents::
+  :local:
+  :depth: 2
+
+Introduction
+------------
+
+Principal component analysis is one technique used to take a large list
+of interconnected variables and choose the ones that best suit a model.
+This process of focusing in on only a few variables is called
+**dimensionality reduction**, and helps reduce complexity.
+
+Motivation
+----------
+
+Dimensionality Reduction
+------------------------
+
+There are two types of dimensionality reduction: feature elimination
+and feature extraction. Feature elimination simply involves pruning
```

Comment:
I would bold "Feature elimination" and "Feature extraction" and probably have white space in between those lines, to emphasize those terms.